src/main/java/seedu/taassist/storage/JsonSerializableTaAssist.java

```diff
@@ -54,6 +61,13 @@ public TaAssist toModelType() throws IllegalValueException {
             }
             taAssist.addStudent(student);
         }
+        for (JsonAdaptedModuleClass jsonAdaptedModuleClass : moduleClasses) {
```

Comment:
I'm concerned with the following scenario:
- Classes has `CS1231S` and `CS2030S` for example
- A student is tagged with class `CS2100`, but it doesn't exist

Whilst it's not behaviour that breaks anything for now, it might be problematic down the line.